code/modules/food_and_drinks/food/snacks_vend.dm

```diff
@@ -178,4 +178,19 @@
 	if(!opened)
 		to_chat(user, span_warning("[src]'s lid hasn't been opened!"))
 		return FALSE
-	return ..()
\ No newline at end of file
+	return ..()
+
+/obj/item/reagent_containers/food/snacks/vermin
+	name = "vermin bites"
+	desc = "A small can with a cartoon mouse on the label. A noise that sounds suspiciously like squeaking can be heard coming from inside."
+	icon_state = "verminbites"
+	tastes = list("rats" = 1 , "mouse" = 2, "cheese" = 1)
+	foodtype = MEAT
+	/// What animal does the snack contain?
+	var/contained_animal = /mob/living/simple_animal/mouse
+
+/obj/item/reagent_containers/food/snacks/vermin/attack_self(mob/user)
+	. = ..()
+	to_chat(user, span_warning("You crack open [src], [contained_animal] falling out from inside!"))
```

Comment:
Why the [contained animal] call if it can only be a mouse? (Unless I'm reading the code wrong)

Reply:
as i understand it if its made to be a var it can be edited with more ease for admins and anyone wanting to code a subtype of the item, for example if an admin wanted to replace the mouse for an ash drake they would only need change the var of the item and they would be able to have an instance of drakes on a can, but i am code iliterate so if im understanding the question wrong please let me know